includes/dart-integrations/firebase-remote-config.mdx

```diff
@@ -0,0 +1,65 @@
+---
+title: Firebase Remote Config
+description: "Learn more about the Sentry Firebase Remote Config integration for the Dart SDK."
+sidebar_order: 6
+platforms:
+  - flutter
+---
+
+The `sentry_firebase_remote_config` integration provides [Firebase Remote Config](https://firebase.google.com/docs/remote-config/get-started?platform=flutter) support for Sentry thus providing insight into feature flag evaluations.
+
+## Behavior
+
+- Adding the `SentryFirebaseRemoteConfig` integration will automatically track feature flag evaluations for boolean Firebase Remote Config values.
+- Only boolean values are supported.
+
+## Prerequisites
+
+1. [Sentry SDK](/platforms/dart/#configure) version `9.0.0` or higher is installed.
+2. Firebase Remote Config is set up.
+
+## Install
+
+To use the `SentryFirebaseRemoteConfig` integration, add the `sentry_firebase_remote_config` dependency.
+
+```yml {filename:pubspec.yaml}
+dependencies:
+  sentry: ^9.0.0
+  sentry_firebase_remote_config: ^9.0.0
+```
+
+## Configure
+
+Add the `SentryFirebaseRemoteConfig` integration to the Sentry SDK initialization.
+
+```dart
+import 'package:sentry_flutter/sentry_flutter.dart';
+import 'package:sentry_firebase_remote_config/sentry_firebase_remote_config.dart';
+
+... // Initialize Firebase Remote Config
+
+await SentryFlutter.init(
+  (options) {
+    options.dsn = 'https://example@sentry.io/example';
+    options.addIntegration(
+      SentryFirebaseRemoteConfigIntegration(
+        firebaseRemoteConfig: firebaseRemoteConfig,
+        // Don't call `await remoteConfig.activate();` when firebase config is updated. Per default this is true.
+        activateOnConfigUpdated: false,
+      ),
+    );
+  },
+);
+```
+
+## Verify & View
+
+### 1. Update Firebase Remote Config value
+
+Update a boolean [Firebase Remote Config](https://firebase.google.com/docs/remote-config/get-started?platform=flutter) value to true. Keep in mind that values will be evaluated as bool if there string values are `true`, `false`, `1`, or `0`. So keep in mind that numerical values of `1` or `0` will be evaluated as `true` or `false` boolean values.
+
+### 2. View the Recorded Feature Flag Evaluation on Sentry.io
+
+To view the recorded feature flag evaluation, log into [sentry.io](https://sentry.io) and open your project.
+
+The recorded feature flag evaluations will be visible in the event context.
```

Comment:
One note for the future: We don't support change tracking for Firebase **yet**.
But once we do, we'll add it to here: https://docs.sentry.io/product/issues/issue-details/feature-flags/#change-tracking

Reply:
Bruno can you clarify - is the wording in the PR ok? because while change tracking is not there, this paragraph talks about [evaluation tracking](https://docs.sentry.io/product/issues/issue-details/feature-flags/#evaluation-tracking)

Reply:
It was just a note for the future. But that makes sense, thanks for pointing out.

It turns out it's not super easy to add Firebase support either and might require the customer to add some firebase lambda in order to call our tracking endpoint. @michellewzhang wrote about it here: https://www.notion.so/sentry/firebase-feature-flag-webhook-1e38b10e4b5d805fb057d2e990ea45e8

I wonder if this is something @aliu39 might be keen to tackle in the future though.